de.yaml

```diff
@@ -84,15 +84,20 @@ months:
     regions: [de_bw, de_by, de_st]
     mday: 6
   3:
-    - name: Internationaler Frauentag
-      regions: [de_be]
-      mday: 8
-      year_ranges:
-        from: 2019
+  - name: Internationaler Frauentag
+    regions: [de_be]
+    mday: 8
+    year_ranges:
+      from: 2019
   5:
   - name: Tag der Arbeit
     regions: [de]
     mday: 1
+  - name: Tag der Befreiung
```

Comment:
I have read wikipedia site and confirmed the 75th anniversary of May 8 2020 is a one-time holiday in Berlin.
Looks good to me.:)